mavenplugin/src/test/java/nl/openweb/tools/Tools.java

```diff
@@ -85,6 +86,9 @@ private static void assertNameCompares(final List<Path> expectedFolderPaths, fin
 
     private static void assertContentCompares(final List<Path> expectedPaths, final List<Path> resultPaths) throws IOException {
         assertEquals(expectedPaths.size(), resultPaths.size());
+//        expectedPaths.sort((left, right) -> left.toUri().getPath().compareTo(right.toUri().getPath()));
```

Comment:
Deze regel mag weg, we hebben Git voor opslag ervan

Reply:
daarnaast, dat is niet meer nodig